package.json

```diff
@@ -38,13 +40,16 @@
   },
   "devDependencies": {
     "babel-cli": "6.7.5",
-    "babel-preset-es2015": "6.6.0",
+    "babel-preset-env": "^1.7.0",
+    "draft-js": ">=0.7.0",
```

Comment:
@TrySound Is there a reason for introducing `draft-js`, `react`, and `react-dom` as dev dependencies here?

Reply:
Tests use them. Peer dependencies are only constraints and yarn do not install them.